defaults/main.yml

```diff
@@ -13,11 +18,11 @@ burp_client_pidfile: "/var/run/burp.pid"
 burp_client_password: "password"
 burp_client_protocol: "1"
 burp_client_syslog: "0"
-burp_client_ca_burp_ca: "/usr/sbin/burp_ca"
-burp_client_ca_csr_dir: "/etc/burp/CA-client"
-burp_client_ssl_cert_ca: "/etc/burp/ssl_cert_ca.pem"
-burp_client_ssl_cert: "/etc/burp/ssl_cert-client.pem"
-burp_client_ssl_key: "/etc/burp/ssl_cert-client.key"
+burp_client_ca_burp_ca: "{{ etc_dir }}/sbin/burp_ca"
```

Comment:
This looks wrong, correct path is in /usr/sbin/burp_ca I think, will confirm and fix on #13 if necessary.

Reply:
this one is definitely as you suggested /usr or /usr/local. I accidentally changed this in my last commit.